.evergreen/config.yml

```diff
@@ -765,6 +789,16 @@ buildvariants:
      - ".4.2"
      - ".4.0"
      - ".3.6"
+- matrix_name: "x509-auth"
```

Comment:
Nothing in particular; I just didn't think there was anything platform-specific about this feature

Reply:
There's not really anything platform specific about most of our features--it's still useful to verify them on other platforms because our dependencies may have platform-specific issues, especially with stuff like TLS. That being said, if it's a lot of work to enable them on Windows / macOS, then it's probably not worth doing. Otherwise, I think we should have full coverage here.

Reply:
Fair enough; I was mostly trying to avoid creating too many new evergreen tasks based on discussions we had before implementing the ticket, but I might have gone too far in the other direction. I'll add the new variants now.

Reply:
unfortunately, it's going to get way worse with OCSP, so I think any hope of keeping a low task count will be dashed soon enough.

Reply:
I mean, if we can get away with only a couple tasks per variant rather than the giant ones we have testing all the server versions, that will be worth something